thirdparty/fetch-thirdparty-deps-windows.sh

```diff
@@ -19,9 +19,9 @@ function get()
 
 if [ ! -f SDL2.dll ]; then
 	echo "Fetching SDL2 from NuGet"
-	get sdl2.redist 2.0.3
-	cp ./sdl2.redist/build/native/bin/Win32/dynamic/SDL2.dll .
-	rm -rf sdl2.redist
+	wget https://www.libsdl.org/release/SDL2-2.0.4-win32-x86.zip
```

Comment:
It's pulling from the official website instead, which is *much* better than a random unverified nuget upload.